packages/backend/src/persistence/storage/postgres/doesAccountExist.ts

```diff
@@ -0,0 +1,17 @@
+import { PrismaClient } from '@prisma/client';
+
+export const doesAccountExist =
+    (db: PrismaClient) => async (ensName: string) => {
+        //Check if account exists
+        const account = await db.account.findFirst({
+            where: {
+                id: ensName,
+            },
+        });
+        //If account exists, return true
+        if (account) {
```

Comment:
return !!account returns a bool indicating if the account is falsey or not

Reply:
done